packages/cozy-stack-client/src/querystring.js

```diff
@@ -1,5 +1,21 @@
 import pickBy from 'lodash/pickBy'
 
+const encodeValues = values => {
+  const encoded =
+    '[' +
+    encodeURIComponent(
+      values
+        .map(value => {
+          return Array.isArray(value)
+            ? '[' + value.map(arrayVal => `"${arrayVal}"`).join(',') + ']'
```

Comment:
The code in `encode` and `encodeValues` is very close, I wonder if we could use recursion here.

Reply:
Probably, but I feel like this should be another PR

Reply:
Why, it's code that is added in this PR ?

Reply:
I checked the code, I misunderstood what you were implying. So ok!